Library/Equipment/Low Tech.eqp

```diff
@@ -12998,7 +12998,7 @@
 		<melee_weapon>
 			<damage>sw-1 cut</damage>
 			<strength>7</strength>
-			<usage>Swung</usage>
+			<usage>Swung - Shortsword</usage>
```

Comment:
Why a qualifier on the usage? It already shows the name of the weapon in the first column of the weapon table... and this isn't a shortsword, anyway, even if one of the skills that can be used for it is.